### DISCUSSION TITLE
Steam Awards

### DISCUSSION THREAD

**Discussion by reanoff** [chr233/ASFEnhance#164] [功能建议 / Ideas]
Steam Awards
https://store.steampowered.com/steamawards/nominations?snr=1_5_9_

**Comment by woctezuma** [comment]
- https://store.steampowered.com/steamawards/nominations
- https://steamcommunity.com/my/badges/65

**Reply by victoronlyarg** [reply]
Is there any command to do it from asf?

**Comment by chr233** [comment]
still testing, will add these functions in the 
next release

**Reply by reanoff** [reply]
There is also a sticker every day in the steam categories (https://store.steampowered.com/category/casual)

**Reply by chr233** [reply]
use `CLAIMITEM` to claim stickers, this cmd can be use in every sale event

**Comment by chr233** [comment]
added in 2.0.1.0

use `VOTE bots <appIds>` to vote for steam award
use `CHECKVOTE` to check badge info

**Reply by woctezuma** [reply]
Thank you!

**Reply by woctezuma** [reply]
There is a small issue: https://github.com/chr233/ASFEnhance/issues/167

**Comment by bolexator** [comment]
It's not working correctly for me at all. If you manually enter IDs, it still may not vote somewhere. The command !vote asf also does not work.
And I do not understand, the script should run the game and change the overview from the voting box? Or it must be done manually?
So far the script works for voting and then not fully.

**Reply by woctezuma** [reply]
> And I do not understand, the script should run the game and change the overview from the voting box? Or it must be done manually?

Probably manually, because the ASF plugin does not know whether you voted for a game which you own. It could know, but that seems too complicated for the task.

**Comment by pontifezo** [comment]
!vote **asf** is not working - it makes votes only for for the first bot

Also plugin votes only in 10 categories, but 11th category "Sit Back and Relax Award" is not voted

**Comment by chr233** [comment]
lol, write a bug in 2.0.1.0, fixed in 2.0.1.1, and add web request limit in 2.0.1.1

**Reply by reanoff** [reply]
what

**Comment by reanoff** [comment]
[ASFEnhance 2.0.1.1](https://github.com/chr233/ASFEnhance/releases/tag/2.0.1.1) Pre-release

**Reply by enot2t** [reply]
VOTE not working for "Nominate a game in each category". Im try:
!VOTE bot
!VOTE bot 440
!VOTE bot [list with appid from 11 games]

Maybe I'm doing something wrong...

**Reply by reanoff** [reply]
!vote asf 1868140,2296380,440,531640,1997040,1260320,1977170,1624540,1976950,1817230,2225070

**Reply by enot2t** [reply]
![photo_2023-11-23_00-14-03](https://github.com/chr233/ASFEnhance/assets/65041683/075ffbd3-bfcc-4b90-8b1a-baf9d6d47fc4)
In the "... in each category", the problem was not solved anyway :(

**Reply by reanoff** [reply]
https://github.com/chr233/ASFEnhance/issues/169

**Comment by chr233** [comment]
fixed in 2.0.1.3 forget to use Url encoding "+", and cause some errors
